src/main/java/gov/cms/mat/cql_elm_translation/utils/HumanReadableUtil.java

```diff
@@ -58,14 +58,24 @@ public static List<String> getMeasureTypes(Measure measure) {
   }
 
   public static String getStratification(Measure measure) {
+    // Collects and returns all stratification descriptions for display
     if (CollectionUtils.isNotEmpty(measure.getGroups())) {
+      String allDescriptions = "";
       for (Group group : measure.getGroups()) {
+        if (allDescriptions.length() > 0
+            && CollectionUtils.isNotEmpty(group.getStratifications())) {
+          allDescriptions += "\n";
```

Comment:
It will be good to have few unit test cases to verify multiple groups and stratifications.